man/systemd.network.xml

```diff
@@ -1741,6 +1741,17 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>AssignAcquiredDelegatedPrefixAddress=</varname></term>
```

Comment:
Is the `Acquired` in the option name providing any value? It is already very verbose. I would probably prefer just `AssignDelegatedPrefixAddress=`.

Reply:
I would prefer to have `Acquired`. That is self explanatory.